eve/workers/openstack-multiple-nodes/terraform/network.tf

```diff
@@ -6,6 +6,17 @@ variable "openstack_network" {
   }
 }
 
+locals {
+  control_plane_network = {
+    name    = "${local.prefix}-control-plane"
+    cidr    = "192.168.1.0/28"
+    gateway = "192.168.1.14"
```

Comment:
It's counter-intuitive to have the gateway address as the last address (before broadcast) in a network, in general it'd be the first, but sure this works :man_shrugging:

Reply:
It's only a detail, but I'll change it if it bother you.